runtime/src/configs.rs

```diff
@@ -448,7 +409,7 @@ impl pallet_utility::Config for Runtime {
     type PalletsOrigin = OriginCaller;
     type RuntimeCall = RuntimeCall;
     type RuntimeEvent = RuntimeEvent;
-    type WeightInfo = weights::pallet_utility::WeightInfo<Runtime>;
+    type WeightInfo = weights::pallet_utility::ZKVEvmWeight<Runtime>;
 }
 
 parameter_types! {
```

Comment:
These are all related to pallet_treasury and can all be removed, no ?

Reply:
Sure, I didn't investigate about what `pallet_utility` was used for.

Reply:
Sorry I am referring to the parameter_types! block with parameters referring to the pallet treasury (line 415)